sourmash/compare.py

```diff
@@ -0,0 +1,230 @@
+import itertools
```

Comment:
please add a docstring at the top of this file - something like 'Functionality for comparing many signatures, used in sourmash compare.'

Reply:
added